floodsub.go

```diff
@@ -54,6 +58,9 @@ type PubSub struct {
 	// topics tracks which topics each of our peers are subscribed to
 	topics map[string]map[peer.ID]struct{}
 
+	// sendMsg handles messages that have been validated
+	sendMsg chan sendReq
```

Comment:
I am thinking that this should be a `chan *sendReq` for consistency with the other channels.